yairl/util/util.py

```diff
@@ -179,40 +128,42 @@ def make_save_policy_callback(savedir, file_prefix, save_interval):
     save_interval (int): The number of training timesteps in between saves.
     """
     step = 0
-    def callback(locals_, globals_):
+
+    def callback(locals_, _):
         nonlocal step
         step += 1
         if step % save_interval == 0:
             policy = locals_['self']
-            # TODO: After we use globs in scripts.data_generate_...,
-            # then we can simply use step.
-            filename = "{}-{}".format(file_prefix, step//save_interval)
+            filename = policy_filename(policy.__class__, policy.get_env(), step)
             save_trained_policy(policy, savedir, filename)
         return True
+
     return callback
 
 
-def load_trained_policy(env, **kwargs):
-    """
-    Load a trained policy from saved_models/.
-    """
-    return load_policy(env, basedir="saved_models", **kwargs)
+def get_policy_paths(env, policy_model_class, basedir, n_experts):
+    assert n_experts > 0
 
+    path = os.path.join(basedir, policy_filename(policy_model_class, env))
+    paths = glob.glob(path)
 
-def load_expert_policy(env, **kwargs):
-    """
-    Load an expert policy from expert_models/.
-    """
-    return load_policy(env, basedir="expert_models", **kwargs)
+    paths.sort(key=lambda f: int(''.join(filter(str.isdigit, f))))
+
+    if len(paths) < n_experts:
+        raise ValueError(
+            """
```

Comment:
I'd favour using implicit line joining as per http://google.github.io/styleguide/pyguide.html#310-strings rather than `"""`.
Alternatively, could use a variable for the string, e.g. `msg = "Wanted to load ..."`, `raise ValueError(msg.format(...))`.

Reply:
Done